### ISSUE TITLE
Troubleshoot gems not allowing Family Letters assets to be precompiled or clobbered

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#204]
I _think_ we're ready to merge the last few accessibility remediation commits, but I can't fully update the dev site to use the newest version of Orchid (`dev` branch for now). When I run the command to precompile assets, I get the following error. Same with the command to clobber assets.

This isn't an accessibility issue persay, but it's blocking the remediation, so I've added the a11y labels to this issue.

``` bash
RAILS_ENV=production SECRET_KEY_BASE=0 rails assets:precompile && RAILS_ENV=production SECRET_KEY_BASE=0 rails assets:clean
/usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support/logger_thread_safe_level.rb:16:in `<module:LoggerThreadSafeLevel>': uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger (NameError)

    Logger::Severity.constants.each do |severity|
    ^^^^^^
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support/logger_thread_safe_level.rb:9:in `<module:ActiveSupport>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support/logger_thread_safe_level.rb:8:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support/logger_silence.rb:5:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support/logger.rb:3:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/activesupport-6.1.7.10/lib/active_support.rb:29:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/railties-6.1.7.10/lib/rails/command.rb:3:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/railties-6.1.7.10/lib/rails/commands.rb:3:in `<main>'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from /usr/local/rvm/gems/ruby-3.1.7@family_letters/gems/bootsnap-1.24.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:33:in `require'
	from bin/rails:5:in `<main>'
```

**Comment by erinchambers** [comment]
I was able to get around this by switching the `concurrent-ruby` version in `Gemfile.lock` to version `1.3.4`. Is that an okay solution to stick with, or is there something else we should do instead?

**Comment by wkdewey** [comment]
I can add `require logger` to config/boot.rb. That has been the solution to similar errors about Logger.

I think it is not considered good practice to pin a dependency to a specific version (it makes it more difficult to update apps), so I try to avoid it if possible.

**Comment by erinchambers** [comment]
Sounds good. and if it's just a "me" problem, we can close this issue

**Comment by techgique** [comment]
https://github.com/CDRH/family_letters/commit/54d158e0e25037febde6fd1311e214e7e1370dec on `dev` branch.
I fixed this for Family Letters recently, so this _should_ be taken care of now. As Will mentioned, any pinning of gems to specific versions like suggested above should be undone. Hopefully that takes care of everything here now.
